### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by Tufahel**
# Description
> In this project, I have created a form of UI for the school library app. This way it can be invoked as an executable.

## General requirements
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

## Ruby requirements
- Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).

## Project requirements

- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+  attr_accessor :date
+  attr_reader :book, :person
 
-  def initialize(book, date, person)
+  def initialize(date, book, person)
     @date = date
 
-    @person = perosn
+    @person = person
     person.rentals << self
```
Comment (uche-inyama) [inline rental.rb:10]: When I try to create a rental, I get an error, which reads that method rental, is not defined in the person class.

For reference, take a look at your book.rb file.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  when '5'
+    create_rental
+  when '6'
+    list_all_rentals
+  else
+    puts 'Please Enter a number between 1 and 7'
+  end
+end
+
+main
```
Comment (uche-inyama) [inline main.rb:39]: It would make your code more readable if you prefix parentheses on a method call.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi @Tufahel :wave:  

Your project is looking good, except for a few minor corrections.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb (hunk 1 above)
Comment (Tufahel) [reply rental.rb:10]: Thanks for your review, I have worked on it.

File: main.rb (hunk 2 above)
Comment (Tufahel) [reply main.rb:39]: @uche-inyama Thanks for your review. I haven't understood your suggestion. Can you please explain it?

Comment (Tufahel) [reply main.rb:39]: In ruby, adding extra parentheses gives a linter error. So, I remove it.
Screenshot from 2022-05-15 02-43-03

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Tufahel

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 2 above)
Comment (uche-inyama) [reply main.rb:39]: Okay.

**Pull request merged**